apps/core/static/js/admin/ForeignKeyLimitChange.js

```diff
@@ -0,0 +1,130 @@
+// // Variables naming convention:
+// //  - names of the jQuery object always start with `$`
+// //  - variables without jQuery functionality don't have `$` sigh in names
+// //  - Exact `$` variable is basic jQuery object
+
+function hideAddButton(objectId) {
+    const addButtonId = "#add_" + objectId;
+    const addButton = $(addButtonId);
+    addButton.css({"display": "none"});
+}
+
+function getLableName(elementId) {
+    // Return lable attribute for select2 <span>
+    return '[aria-labelledby="select2-' + elementId + '-container"]';
+}
+
+function disableField($object) {
```

Comment:
```suggestion
function disableField($field) {
```
Так же и с кнопками лучше поступить, если известно что конкретно будет дизейблиться